### DISCUSSION TITLE
Ram requirements

### DISCUSSION THREAD

**Discussion by jaydenchan1027** [upscayl/upscayl#645] [Q&A] [answered]
What is the requirements of ram in linux

**Comment by aaronliu0130** [comment]
I don't know an exact number, but 8gb and a GPU should be more than enough to run it

**Reply by Wyrdgirn** [reply, marked as answer]
On Windows it works with a minimum of 2GB of RAM and 2GB of pagefile on computers without GPU with a 326x235 and 270x480 images with x4 scaling using 80-98% of RAM (46-51% used by Windows) without problems, in Linux it may be similar since it manages memory better, at least that's how it worked for me. With GPU it may be similar or better, idk...
